### DISCUSSION TITLE
Return all occurences from the search box

### DISCUSSION THREAD

**Discussion by AlbatrosHurleur** [GMOD/jbrowse-components#3166] [Ideas]
Hello,

There are 3 genes in my genome annotation that share the same name. This particular GFF3 track is indexed on Name,ID,description,classification and when I search by gene name only the first gene is shown in the results.

I am not familiar with trix, but I had a look in the ix file and it seems that my search pattern is indeed associated with the 3 genes.

Would it be feasible on your side to return all occurrences of a search results, maybe suffixed by something like ***(match #n)*** in order to discriminate them from each other?

Thanks!

**Comment by cmdcolin** [comment]
Currently, the list of features in drop down is "de-duplicated" to avoid confusion of multiple identically named things in the list. But, it is programmed so that if there are multiple features corresponding to a single name, it will pop up a dialog box when this occurs. 

If you aren't getting a dialog box or if you want something different from a dialog box let me know...

**Comment by AlbatrosHurleur** [comment]
A dialog box seems perfect! But indeed I am not getting one (v2.0.1 on Firefox and Chrome).
Is it supposed to pop up after clicking, in the drop-down list, on the term having multiple features? If so, on my side I directly jump to the first gene in the index.

The line having the term in the first field of the ix file is like this:
`term ["location1"|"trackname"|"name"|"ID1"|"description1"],1 ["location2"|"trackname"|"name"|"ID2"|"description2"],2 ["location3"|"trackname"|"name"|"ID3"|"description3"],3`

What should I check to see where the issue is coming from?

Thanks!

**Reply by cmdcolin** [reply]
Oh right I just saw this also, the dialog box is not coming up. I'll try to diagnose

**Comment by AlbatrosHurleur** [comment]
Hi,

I just saw this was fixed in the latest release, it works great! Thanks!
